tests/test_sigma_cli_integration.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+set -e
+
+plugin="loki"
```

Comment:
Couldn't we get the plugin as an argument to the script instead of hardcoding it?

Reply:
Yes and no. Properly supporting parameterized backends would require some more serious changes to the first call to `poetry install` (since that currently installs the current working project) - which I didn't really want to do (I don't think it's possible from poetry, so would have to use `pip`, which would defeat the point of this integration test honestly!) - but I also wanted to make the script easily transferable to other backends going forward (potentially so it could be added to the default pySigma backend template).

Reply:
Ok, that makes perfect sense to me, thanks for explaining 👍